### PR TITLE
(fix) you can create an event without airtable an airtable ID or partner url

### PR DESCRIPTION
**Pull request by tmb**
closes #268

### CODE REVIEW

**Review by thesephist — APPROVED** [review]
lgtm

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -16,6 +16,9 @@ def new
 
   # POST /events
   def create
+    event_params[:club_airtable_id] = nil if event_params[:club_airtable_id].empty?
```
Comment (lachlanjc) [inline app/controllers/events_controller.rb]: Not positive, but isn’t this a good situation to use `||=`?
```suggestion
    event_params[:club_airtable_id] ||= nil
```
Definitely check that that’s correct/works

& _super_ not sure about this, but couldn’t you then assign both on the same line, comma-separated?

Comment (thesephist) [reply app/controllers/events_controller.rb]: I don't think `.empty?` is equivalent to whatever `||=` checks for -- I personally prefer being more explicit here.

Not 100% sure but I believe `||=` only works for `nil` and `false`? http://www.rubyinside.com/what-rubys-double-pipe-or-equals-really-does-5488.html

Comment (lachlanjc) [reply app/controllers/events_controller.rb]: Ok, definitely go for whatever you want. To me, seems overly verbose for such simple functionality. Maybe a `merge`/`reverse_merge`?

**Pull request merged**